OWNERS_ALIASES

```diff
@@ -1,12 +1,22 @@
 # See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
```

Comment:
We should delete this whole `OWNERS_ALIASES` file as we're not using aliases in this repo.  It will just be something that gets out of date and isn't maintained.